pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -69,8 +69,8 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		projectDisplayName = projectRequest.DisplayName
 	}
 
-	if len(projectRequest.Annotations["description"]) > 0 {
-		projectDescription = projectRequest.Annotations["description"]
+	if len(projectRequest.Annotations[projectapi.ProjectDescription]) > 0 {
```

Comment:
Make sure you can create a project using the three methods and set the display name and description:
`osadm new-project`, `osc new-project`, and the web UI